util/math/math_test.go

```diff
@@ -48,3 +48,24 @@ func TestStrLenOfUint64Fast(t *testing.T) {
 		}
 	})
 }
+
+func TestMinInt(t *testing.T) {
+	require.Equal(t, 1, Min(1, 2))
+	require.Equal(t, 1, Min(2, 1))
+	require.Equal(t, 1, Min(4, 2, 1, 3))
+	require.Equal(t, 1, Min(1, 1))
+}
+
+func TestMaxInt(t *testing.T) {
+	require.Equal(t, 2, Max(1, 2))
+	require.Equal(t, 2, Max(2, 1))
+	require.Equal(t, 4, Max(4, 2, 1, 3))
+	require.Equal(t, 1, Max(1, 1))
+}
+
+func TestMinInt64(t *testing.T) {
+	require.Equal(t, 1, Min(1, 2))
+	require.Equal(t, 1, Min(2, 1))
+	require.Equal(t, 1, Min(4, 2, 1, 3))
+	require.Equal(t, 1, Min(1, 1))
+}
```

Comment:
Is this really testing Int64? In that case what is the TestMinInt testing ;)
Maybe have tests for all different int types and why not also for other constraints.Ordered types, like float and string types?

Reply:
This is indeed a problem caused by migration, let me refactor it.

Reply:
Put all tests of Max/Min into a test case.